app/CommandExecutor.php

```diff
@@ -13,6 +13,21 @@ class CommandExecutor
     /** @var array $commands */
     protected $commands = [];
 
+    /** @var string $outputFileName */
+    protected $outputFileName = '';
+
+    public function __construct()
+    {
+        $this->outputFileName = @tempnam(sys_get_temp_dir(), 'fwd_output_');
```

Comment:
Do we need the `@`? I guess in case of any error is raised, we might wanna stop and not ignore it, right?

Reply:
Also very important - do we want to have a single output buffer file? If we have more than one command run quietly, we need to make sure to overwrite the contents of if so output is not mixed among the commands executed.

Reply:
`tempnam` raises a notice even in case of success. `@` is here to silent it. https://bugs.php.net/bug.php?id=69489

Reply:
well, maybe we could just make our own file path and `file_put_contents` and empty string